README.md

```diff
@@ -426,6 +428,35 @@ for (let item of evenly(set1, set2)) {
 // ➜ 1 3 2 4
 ```
 
+### reverse(iterator)
+
+Returns an reversed Iterator.
+
+**Important:** don't use infinite iterator, reverse method fall into endless loop.
```

Comment:
don't use infinite iterator, reverse method fall into endless loop -> don't use infinite iterator, otherwise `reverse` method will fall into endless loop